Decred Wallet/Features/TransactionDetails/Cells/TransactionDetailCell.swift

```diff
@@ -1,38 +1,47 @@
 //  TransactionDetailCell.swift
 //  Decred Wallet
 //
-// Copyright (c) 2018-2019 The Decred developers
+// Copyright (c) 2018-2020 The Decred developers
 // Use of this source code is governed by an ISC
 // license that can be found in the LICENSE file.
 
 import UIKit
 
+struct TransactionDetails {
+    let title: String
+    let value: NSAttributedString
+    let isCopyEnabled: Bool
+}
+
 class TransactionDetailCell: UITableViewCell {
-    
-    @IBOutlet private weak var labelTitle: UILabel!
-    @IBOutlet private weak var labelValue: UILabel!
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var valueButton: UIButton!
+    var presentingController: TransactionDetailsViewController?
```

Comment:
2 things.
First, there should be an empty line space separating this declaration from the storyboard outlets.
Second, looking at how this variable is used, I don't think we need to pass in an entire VC. A simple callback would do:

```suggestion

    var onTxDetailValueCopied: ((_ copiedDetail: String) -> ())?
```